examples/asr/asr_adapters/train_asr_adapter.py

```diff
@@ -168,6 +175,9 @@ def main(cfg):
     # Then, Unfreeze just the adapter weights that were enabled above (no part of encoder/decoder/joint/etc)
     model.unfreeze_enabled_adapters()
 
+    # Update model config prior to training
+    model.cfg = model.cfg
+
     # Finally, train model
     trainer.fit(model)
```

Comment:
Maybe add adapter-only saving/loading to this file

Reply:
Good point, forgot about that.

Reply:
Done